common/scala/src/main/resources/application.conf

```diff
@@ -12,3 +12,22 @@ akka.http {
         max-open-requests = 128
     }
 }
+
+akka.tracing {
+    #zipkin configuration
+    host = "localhost"
```

Comment:
all of these settings imply scribe, which is a deprecated transport. most start with http and name the config endpoint ex https://github.com/openzipkin/zipkin-reporter-java/blob/master/okhttp3/src/main/java/zipkin/reporter/okhttp3/OkHttpSender.java#L73